EIPS/eip-5496.md

```diff
@@ -76,7 +76,7 @@ interface IERC721Cloneable {
     /// @param tokenId The identifier of the queried NFT
     /// @param privilegeId The identifier of the queried privilege
     /// @param referrer The address of the referrer
-    /// @return Whether the operation is sucessful or not
+    /// @return Whether the operation is successful or not
```

Comment:
Could you aslo specify what will happen to the "privileges" if a token is transfered?

Reply:
If the owner of the EIP-5496 NFT transfer ownership to another user, there is no impact on "privileges". But errors may be occurred when the owner tries to withdraw the original EIP-721 token from EIP-5496 via calling function unwrap() in our implementation contract if any available privilege is still ongoing. We protect the rights of holders of the privileges by checking the last expire date of the privilege.
  
function unwrap(uint256 tokenId, address to) external {
        require(getBlockTimestamp() >= privilegeBook[tokenId].lastExpiresAt, "privilege not yet expired");
        require(ownerOf(tokenId) == msg.sender, "not owner");
        _burn(tokenId);
        IERC721(nft).transferFrom(address(this), to, tokenId);
        emit Unwrap(nft, tokenId, msg.sender, to);
    }